deploy/playbooks/roles/common/templates/prod.py.j2

```diff
@@ -63,6 +63,10 @@ logging = {
 # graphite/statsd settings
 graphite_api_key = "{{ graphite_api_key }}"
 
+# callback url
+callback_url = "{{ callback_url|default('') }}"
```

Comment:
Perhaps we should put these settings in a ``prod_callbacks.py`` like we do with the db connection and api creds?

That would allow us to deploy without defining these after they've already been set.